streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -56,28 +58,35 @@
     private static final String TOPIC = "topic";
     private final KStreamBuilder builder = new KStreamBuilder();
     private KGroupedStream<String, String> groupedStream;
+    private KStreamTestDriver driver = null;
 
     @Before
     public void before() {
         final KStream<String, String> stream = builder.stream(Serdes.String(), Serdes.String(), TOPIC);
         groupedStream = stream.groupByKey(Serdes.String(), Serdes.String());
     }
 
+    @After
+    public void cleanup() {
```

Comment:
Personally not a fan of introducing base classes for testing. IMO it would be better to change `KStreamTestDriver` to extend `org.junit.rules.ExternalResource`. It can override the `after` method and call close there.